cmd/download.go

```diff
@@ -197,6 +208,7 @@ type downloadPayload struct {
 func initDownloadCmd() {
 	downloadCmd.Flags().StringP("uuid", "u", "", "the solution UUID")
 	downloadCmd.Flags().StringP("track", "t", "", "the track ID")
+	downloadCmd.Flags().StringP("token", "k", "", "authentication token used to connect to exercism.io")
```

Comment:
Dang. I totally see why this is `k`, but now I'm wracking my brains trying to figure out if we should rename the whole `token` concept so we can have a consistent and unambiguous set of flags.

Reply:
i liked when it was called `key`

Reply:
Yeah, we might need to go back to that. Meanwhile, we'll keep this and re-evaluate before we launch v2. Thank you!